### ISSUE TITLE
pretrain learning rate is le-8?

### ISSUE THREAD

**Issue by hegang1-tal** [baichuan-inc/Baichuan-7B#119]
### Required prerequisites

- [X] I have read the documentation <https://github.com/baichuan-inc/baichuan-7B/blob/HEAD/README.md>.
- [X] I have searched the [Issue Tracker](https://github.com/baichuan-inc/baichuan-7B/issues) and [Discussions](https://github.com/baichuan-inc/baichuan-7B/discussions) that this hasn't already been reported. (+1 or comment there if it has.)
- [X] Consider asking first in a [Discussion](https://github.com/baichuan-inc/baichuan-7B/discussions/new).

### Questions

hi, I found in deepspeed config file that the learning rate is le-8.  I am wandering that is this too small for pretraining ?

### Checklist

- [X] I have provided all relevant and necessary information above.
- [X] I have chosen a suitable title for this issue.
